tests/dynamic_data/publications/map_layer_relation/map_internal_layer_referenced_by_workspace_name_test.py

```diff
@@ -0,0 +1,57 @@
+import os
+import pytest
+
+from layman import LaymanError
+from test_tools import process_client
+from tests import EnumTestTypes, Publication4Test
+from tests.dynamic_data import base_test, base_test_classes
+
+
+DIRECTORY = os.path.dirname(os.path.abspath(__file__))
+
+
+WORKSPACE = 'test_map_internal_layer_by_ws_name'
+INTERNAL_LAYER = Publication4Test(WORKSPACE, process_client.LAYER_TYPE, 'internal_layer', uuid='a0939eb1-3c48-4ad6-949e-0a170419ba90')
+
+
+TEST_CASES = {
+    'one_internal_layer': {
```

Comment:
I suggest to use more internal layers referenced by name in one test case. We can add e.g.
- WMS layer with global GS workspace and workspace in LAYERS parameter
- WFS layer

Reply:
Second test case added.